common/djangoapps/contentserver/test/test_contentserver.py

```diff
@@ -32,6 +33,10 @@
 TEST_DATA_DIR = settings.COMMON_TEST_DATA_ROOT
 
 
+def get_versioned_asset_url(url):
+    return '/assets/' + md5.new(url).hexdigest() + '/' + url
```

Comment:
Correct.  I just needed a way to generate an md5'd version of an asset path.  The hash doesn't even need to be unique across assets.  The hash is just there for browsers to cachebust on.  On production, they'll be real hashes based on the content.  In tests, we just need to match the right URL pattern to hit the codepaths.